code/game/objects/items.dm

```diff
@@ -149,12 +152,23 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		if(damtype == "brute")
 			hitsound = "swing_hit"
 
-	if (!embedding)
-		embedding = getEmbeddingBehavior()
-	else if (islist(embedding))
-		embedding = getEmbeddingBehavior(arglist(embedding))
-	else if (!istype(embedding, /datum/embedding_behavior))
-		stack_trace("Invalid type [embedding.type] found in .embedding during /obj/item Initialize()")
+	if(embedding)
+		var/list/temp = embedding
+		temp.Insert(0, /datum/element/embed) // i dunno about this
+		AddElement(arglist(temp))

```

Comment:
You could just pass the parameters as list, these are unique by id so bespoke will get same ref anyway.